resources/js/visual-editor/site-editor/site-editor-app.css

```
@@ -0,0 +1,143 @@
+/**
+ * Site-editor shell layout.
+ *
+ * Mirrors the post-editor shell (`editor/editor-app.css`) so the two
+ * editors feel like siblings (P3 in the macro design brief). Differences:
+ *   - Uses `--ap-site-editor-*` tokens namespaced separately so themes
+ *     can tune each surface independently.
+ *   - Renders the navigator on the left where the post editor renders
+ *     the inserter; the inspector keeps its right-rail position.
+ */
+
+@import url('../editor/top-bar.css');
```

Stylelint (17.9.0)

[error] Expected "url('../editor/top-bar.css')" to be "'../editor/top-bar.css'" (import-notation)

(import-notation)